test/coordinate_unit_test.js

```diff
@@ -0,0 +1,56 @@
+/* global describe, it */
```

Comment:
Nice to have, but I think this should move in a new PR.

Or if nobody is against it, you could reword the PR title / commits to move from a 'fix bug' PR to a 'feature: add Coordinates conversion unit testing' PR.

Reply:
> Nice to have, but I think this should move in a new PR.

As it is testing the new hopefully bug-free behaviour, why do you want it in another PR?

> feature: add Coordinates conversion unit testing

a unit test is not a feature imo. I'm not against separating the bug fix from the test in 2 commits (although I'd prefer one commit), but it should be prefixed with 'chore' (or 'test' as it is currently), right?